src/index.js

```diff
@@ -0,0 +1,19 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import _ from 'lodash';
```

Comment:
Hello, thanks for the work done so far, but I propose that lodash is removed as is not required as of yet.